code/datums/controllers/action_controls.dm

```diff
@@ -395,6 +395,8 @@ var/datum/action_controller/actions
 			src.proc_path = proc_path
 		else //no proc, dont do the thing
 			CRASH("no proc was specified to be called once the action bar ends")
+		if(proc_args)
```

Comment:
On topic of quick last-minute consistency fixes, with the rest of the proc, because why not.

```suggestion
		if (proc_args)
```